lametro/views.py

```diff
@@ -209,10 +221,37 @@ def get_context_data(self, **kwargs):
         except EventDocument.DoesNotExist:
             pass
 
+        latest_action = BillAction.objects.filter(bill=OuterRef("pk")).order_by(
+            "-order"
+        )
+
+        related_bills = (
+            LAMetroBill.objects.filter(eventrelatedentity__agenda_item__event=event)
+            .defer("extras")
+            .prefetch_related(
+                Prefetch(
+                    "versions",
+                    queryset=BillVersion.objects.filter(
+                        note="Board Report"
+                    ).prefetch_related("links"),
+                    to_attr="br",
+                ),
+                "packet",
+            )
+            .annotate(
```

Comment:
This annotation is meant to bypass the many queries in `LAMetroBill.inferred_status` to get the last action for each bill. 

@fgregg mentioned that it might be a good idea to bypass `LAMetroBill.inferred_status` in all other views/templates by adding this annotation to the standard `LAMetroBill` manager.

Reply:
What are your thoughts on replacing it across the board?

Reply:
I think it might be nice to have it accessible when requested, so it's not baked into `objects`, but you can get at it easily, like `LAMetroEvent.objects.with_media`: https://github.com/Metro-Records/la-metro-councilmatic/blob/a63e3359080b818de6df932321ca406f9086a4a9/lametro/models.py#L511

Reply:
Yeah, I like adding it as an optional method on the `LAMetroBill` manager.

We use the inferred status property on some board report detail pages so we should probably keep it for that. It's also used in search faceting but optimizing that might require some more thought.